image/wmf.ksy

```diff
@@ -252,7 +252,7 @@ enums:
     0x0003: masknotpen
     0x0004: notcopypen
     0x0005: maskpennot
-    0x0006: not
+    0x0006: not_
```

Comment:
This one won't really work well. This is currently a problem with compiler with allows that, but in reality, for languages that use lowerCamelCase, this `not_` will still be rendered as `not`, so, ideally, we should prohibit trailing / extra underscores in identifiers :(

Feel free to rename with some sort of prefix?

Reply:
hehe, _not?

But I think a better solution is that the backend for each language has an identifier filter that transforms any reserved words into legal words.

Reply:
`_not` would result in the same, and, moreover, we've reserved all `_` prefixed identifiers for internal KS stuff (like `_root`, `_parent`, `_index`, etc).

Yeah, https://github.com/kaitai-io/kaitai_struct/issues/90 would have helped...